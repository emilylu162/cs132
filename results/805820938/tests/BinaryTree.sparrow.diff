18,19c18,26
< --- Error (Explicit) ---
< null pointer
---
> 4
> 8
> 14
> 16
> 20
> 24
> 28
> 0
> 0
