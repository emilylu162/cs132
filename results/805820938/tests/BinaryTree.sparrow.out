16
100000000
8
16
4
8
12
14
16
20
24
28
1
1
1
0
1
--- Error (Explicit) ---
null pointer
